src/daemon/daemon.cpp

```diff
@@ -2916,6 +2916,22 @@ void mp::Daemon::create_vm(const CreateRequest* request,
     // TODO: We should only need to query the Blueprint Provider once for all info, so this (and timeout below) will
     //       need a refactoring to do so.
     const std::string blueprint_name = config->blueprint_provider->name_from_blueprint(request->image());
+    // if the launch target is a blueprint instead of an image, issues a deprecation warning
+    if (!blueprint_name.empty())
+    {
+        constexpr auto deprecation_warning =
+            "* Warning! Blueprints are deprecated and will be removed in a future release. *\n\n"
+            "You can achieve similar effects with cloud-init and other launch options.\n"
+            "Run `multipass help launch` for help, or find out more at:\n"
+            "- "
+            "https://documentation.ubuntu.com/multipass/en/latest/how-to-guides/manage-instances/"
+            "launch-customized-virtual-machines-with-multipass-and-cloud-init/\n"
+            "- https://cloudinit.readthedocs.io/en/latest/index.html\n\n";
```

Comment:
```suggestion
            "- https://cloudinit.readthedocs.io\n\n";
```
Remove localization so the user's browser will localize the page.